guides/common/modules/con_preparing-for-disaster-recovery.adoc

```diff
@@ -0,0 +1,5 @@
+[id="preparing-for-disaster-recovery"]
+= Preparing for disaster recovery
+
+{Team} recommends preparing a disaster recovery plan to ensure the continuity of {Project} services in case of a disruptive event.
```

Comment:
Should this be `The {Team}`?

Reply:
This is how it's rendered in the affected flavors:

- katello: Foreman community recommends
- orcharino: ATIX AG recommends
- satellite: Red Hat recommends

It looks like we are using `{Team} recommends` at quite a few other places as well. I'm not sharing this to prove that it's okay, just to say that if it's a problem, it's a bigger one and we probably don't need to deal with it here :) So, no change.